lib/database/leaderboard.php

```diff
@@ -577,12 +589,29 @@ function GetFormattedLeaderboardEntry($formatType, $scoreIn)
         settype($mins, 'integer');
         settype($secs, 'integer');
         settype($milli, 'integer');
-        return sprintf("%01d:%02d:%02d.%02d", $hours, $mins, $secs, $milli);
-    } else {
+        return sprintf("%s.%02d", _FormatValueSeconds($hours, $mins, $secs), $milli);
+    } elseif ($formatType == 'MINUTES') { // Number of minutes
+        $hours = $scoreIn / 60;
+        settype($hours, 'integer');
+        $mins = $scoreIn % 60;
+        return sprintf("%01dh%02d", $hours, $mins);
+    } elseif ($formatType == 'SCORE') { // Number padded to six digits
+        return sprintf("%06d", $scoreIn);
+    } else { // Raw number
         return "$scoreIn";
     }
 }
 
+function isValidLeaderboardFormat($formatType)
+{
+    return $formatType == 'TIME' ||      // Frames
+           $formatType == 'TIMESECS' ||  // Seconds
+           $formatType == 'MINUTES' ||   // Minutes
+           $formatType == 'MILLISECS' || // Hundredths of seconds
+           $formatType == 'VALUE' ||     // Raw number
+           $formatType == 'SCORE';      // Number padded to six digits
```

Comment:
Alternatively:
```
return in_array($formatType, [
    'TIME', // Frames
    'TIMESECS', // Seconds
    'MINUTES', // Minutes
    'MILLISECS', // Hundredths of seconds
    'VALUE', // Raw number
    'SCORE', // Number padded to six digits
]);
```
Seeing those "enums" being used more often in code they could be moved to (class) constants instead of comparing strings.
None of the above is necessary - just as a reminder.